Tests/NIOFSTests/Internal/SyscallTests.swift

```diff
@@ -372,7 +373,7 @@ final class SyscallTests: XCTestCase {
         throw XCTSkip("'sendfile' is only supported on Linux")
         #endif
     }
-
+#endif
```

Comment:
Please move this check into the actual test code so that it is appropriately XCTSkipped.